scripts/build-search-index.js

```diff
@@ -32,7 +31,6 @@ stdin.on("end", () => {
                 this.add({
                     ref: url + "|" + page.title,
                     title: page.title,
-                    content: page.content
```

Comment:
In some ways this may actually make the search experience better (smaller download, faster results), but in other ways makes it worse (not actually indexing content) as you called out.

I am wondering if we should just burn the current search to the ground and change it so that when you land on `/docs/search/?q=whatever` we grab the query parameter and immediately redirect to Google search? (Note: landing on `/docs/search/` first _before_ redirecting, so we can still get logs of searches people are making). That feels like a cheap fix in the meantime until we get around to building out the actual search experience we want, and will improve both build times for us and search experience for users. And could be done quickly as a follow-up PR.

Reply:
Yep, I'm with you. I stopped short of actually doing that as part of this PR (though I almost did!), as the goal here was more just to "stop the bleeding" and unblock the pipeline. I believe @leezen has some thoughts on what the next steps for search might be as well. I'll make sure to bring this up at the next team meeting.